### PR TITLE
Extension methods

### PR DESCRIPTION
**Pull request by DorCoMaNdO**
Extension methods to the Client and the Account classes, that allows to restrict the access to the actual lists/dictionaries and the methods will acquire the data from those lists safely (locks, etc).

I didn't test it, but unless I caused some overflow I believe it should work just fine.

### CODE REVIEW

File: AdminTools/MainFilterscript.cs [hunk 1]
```diff
@@ -537,6 +531,84 @@ public class Account
...
+                TimeIssued = DateTime.Now,
+                Name = client.Name,
+            };
+
+            lock (Lists._banned.BannedIps) Lists._banned.BannedIps.Add(ban);
+
+            client.GetAccount().Ban = ban;
+        }
+
+        public static Client GetClient(this Account account)
```
Comment (Guad) [inline AdminTools/MainFilterscript.cs]: This will always return null, won't it?

File: AdminTools/MainFilterscript.cs [hunk 2]
```diff
@@ -537,6 +531,84 @@ public class Account
...
+        {
+            Client client = null;
+            lock (Program.ServerInstance.Clients) Program.ServerInstance.Clients.Any(c =>
+            {
+                if (c.Name == account.Name) return true;
+
+                return false;
+            });
+
+            return client;
```
Comment (Guad) [inline AdminTools/MainFilterscript.cs]: This will always return null, won't it?

**Comment by DorCoMaNdO** [comment]
Fixed.

**Comment by Guad** [comment]
Looks good!

**Pull request merged**